.github/workflows/renovate.yaml

```diff
@@ -13,7 +13,7 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v4
       - name: Self-hosted Renovate
-        uses: renovatebot/github-action@v39.2.4
+        uses: renovatebot/github-action@v40
```

Comment:
It doesn't require a minor and patch version? I thought it did

Reply:
I dont think so, the action above also has just major version

Reply:
That's not enough, that depends on what each GitHub action has released, and different actions have different releases
Also it doesn't work: https://github.com/stackitcloud/stackit-sdk-go/actions/runs/7744970877/job/21119926497